ddtrace/tracer/tracer.go

```diff
@@ -729,6 +734,14 @@ func (t *tracer) Inject(ctx ddtrace.SpanContext, carrier interface{}) error {
 	if !t.config.enabled.current {
 		return nil
 	}
+
+	if t.config.tracingAsTransport {
+		// in tracing as transport mode, only propagate when there is an upstream appsec event
```

Comment:
```suggestion
		// in tracing as transport mode, only propagate when there is an upstream appsec event
		// TODO: replace with _dd.p.ts in the next iteration standardizing this for other products, comparing enabled products in `t.config` with their corresponding `_dd.p.ts` bitfields
```

Reply:
Applied in https://github.com/DataDog/dd-trace-go/pull/3033/commits/2be912cbe48a65d29cbee2f0699b41ef8e34e0bc